src/pydata_sphinx_theme/assets/styles/components/_search.scss

```diff
@@ -39,46 +56,36 @@
     &::-webkit-search-decoration {
       appearance: none;
     }
+
+    &:focus,
+    &:focus-visible {
+      color: var(--pst-color-text-muted);
+    }
   }
 
   // Shows off the keyboard shortcuts for the button
   .search-button__kbd-shortcut {
     display: flex;
-    position: absolute;
-    right: 0.5rem;
+    margin-inline-end: 0.5rem;
     color: var(--pst-color-border);
   }
 }
 
-.form-control {
```

Comment:
I merged these rules in with the `input` selector above. As far as I can tell, I search the code base, and the only place I could find this class being used is on the input element in the search-field.html component.